server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -811,7 +813,12 @@ private static void mutableAttributes(ConfigurationElement<?> element, Map<Strin
       prefix = prefix == null ? "" : element.elementName();
       for (Attribute<?> attribute : element.attributes().attributes()) {
          if (!attribute.isImmutable()) {
-            attributes.put(prefix + "." + attribute.getAttributeDefinition().name(), attribute);
+            AttributeDefinition<?> definition = attribute.getAttributeDefinition();
+            // even if mutable, we don't want expose this attribute to the user
+            if (AbstractTypedPropertiesConfiguration.PROPERTIES.equals(definition)) {
```

Comment:
I think the best thing to do is to completely remove `PROPERTIES` from the `IndexingConfiguration` and `QueryConfiguration` AttributeSets. leave the methods on the builders but make them log a warning and do nothing

Reply:
I would definitely like the idea of removing these properties. At the moment they are only used by our tests I think. Maybe we can do it in another issue, to have a better track of our changes. What do you think @tristantarrant?

Reply:
Created https://issues.redhat.com/browse/ISPN-14575 thanks